internal/registry/registry.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote/transport"
 )
 
-var Endpoint = "kube-image-keeper-service:5000"
+var Endpoint = "kuik-registry:5000"
```

Comment:
It is the service used by the controller to push images ? So we may also hardcode the Service name in `service.yaml` ?

Reply:
This is set by the flag `--registry-endpoint` defined in `cmd/cache/main.go`, so this value is actually never used. It is set only because some tests fails if it is not. We could set it during tests only and keep it empty here if you prefer.